src/signals/incident/components/form/FileInput/index.js

```diff
@@ -69,7 +71,9 @@ const FileInput = ({
     }
   };
 
-  const checkFileSize = file => file.size <= meta.maxFileSize;
+  const checkMinFileSize = file => file.size >= meta.minFileSize;
+
+  const checkMaxFileSize = file => file.size < meta.maxFileSize;
```

Comment:
NOTE: Updated this check. It used to test `file.size <= meta.maxFileSize` but that would allow files to be equal to maxFileSize while `/src/signals/incident/services/custom-validators/index.js` does not allow for that.